module_utils/kafka_lib_acl.py

```diff
@@ -32,7 +32,7 @@ def process_module_acl(module):
         'acl_resource_type': params['acl_resource_type'],
         'state': params['state']
     }]
-
+    params['mark_others_as_absent'] = False
```

Comment:
`mark_others_as_absent` will always be `False` when calling from kafka_acl module (@see line 43)